imagetest/test_suites/storageperf/setup.go

```diff
@@ -116,24 +169,18 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 	return nil
 }
 
-// Due to compatability issues or other one off errors, some combinations of machine types and images must be skipped. This function returns true for those combinations.
-func skipMachineTypeImage(machineType, image string) bool {
-	if strings.HasPrefix(machineType, "c3d") && (strings.Contains(image, "windows-2012") || strings.Contains(image, "windows-2016")) {
+func skipTest(tc storagePerfTest, image string) bool {
+	if tc.arch == "ARM64" && !strings.Contains(image, "arm64") {
```

Comment:
The goal is to disable arm64 tests at all?

Reply:
No, just to avoid running non-arm test configurations on arm images and vice versa.